### ISSUE TITLE
A11y_Azure data studio_Executed query_Clear All history_Screenreader: Screen Reader is not reading out the status message "No Query to display" After when user activated Clear all history button presents on right navigation using tab key.

### ISSUE THREAD

**Issue by Jtith** [microsoft/azuredatastudio#20891]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #A11yWCAG2.1; #SH_SQLAzureDataStudio_Win32_Oct2022; #SQL Azure Data Studio; #WCAG4.1.3; #Screen reader; #NVDA; #JAWS; #Win11; #FTP; #DesktopApp; #Win32; #NonBenchmark; #Status Message;

### Environment Details:
Application Name: SQL Azure Data Studio
Application Version: 1.40.0- insider
Windows 11 Version: 22621.674
NVDA Version 2020.4
JAWS Version 

### Repro Steps: -

1. Install Azure Data Studio – Insiders
2. Tab till New connection and press enter.
3. Using valid credentials, connect to server.
4. Tab till extensions and press enter.
5. Tab till search box and search "Query History". 
6. Tab till Install and press enter.
7. Tab till new query and press enter
8. Tab till query editor and execute any query.
9. Open Toggle panel using Ctrl+J.
10. Tab till Query History tab and press enter.
12. Tab till to Clear history button present on right navigation
13. Press enter on Clear history to activate.
14. You will observe that screen reader is not announcing the error message of "No Query to display" After when user activated Clear all history button.

### Actual:
Screen Reader is not reading out the status message of "No Query to display" After when user activated Clear all history button presents on right navigation using tab key.

### Expected:
Screen Reader should read out the status message of "No Query to display" After when user activated Clear all history button presents on right navigation using tab key 

### User Impact:
Screen Readers users may not be aware of the status message "No Query to display" after user clear the history.

### Attachment: -
![Screen-Reader-is-not-reading-out](https://user-images.githubusercontent.com/100552367/196868236-240753e8-9a9b-4410-bd1e-1f5ace3551eb.gif)

**Comment by Charles-Gagnon** [comment]
@Jtith That's not a "status" message - it's meant to show that there's specifically no entries in the list. The user already knows they clicked the "clear all" button so do we really need to announce to them again that there aren't any items present?

**Comment by Charles-Gagnon** [comment]
@Jtith Could you please respond to the above question? I want to be sure of what the expectation is here before I change anything here since it seems to be working as intended to me

**Comment by Jtith** [comment]
@Charles-Gagnon As per observation in the environment. after we press  enter on "clear all" button, screenreader is not announcing 0 items in list and Also, there is a message popping in the screen "No Query to display" is also, not being announced.  "No Query to display" should be announced to fix this issue.

**Comment by Charles-Gagnon** [comment]
Per office hours this is being downgraded to a sev3

**Comment by Jtith** [comment]
As per guidance Sev3 bugs don't have any fixed SLA so you can fix or add it in your backlog

**Comment by InduPriya1805** [comment]
GitHubTags:#SH-SQL Azure Data Studio-Win32-OCT2022
